backend/alembic/migration.py

```diff
@@ -0,0 +1,16 @@
+from alembic.config import Config
+from alembic import command
+import inspect
+import os
+
+
+def main(event, context):
+    _current_dir = os.path.dirname(os.path.abspath(inspect.stack()[0][1]))
```

Comment:
```suggestion
    _current_dir = os.getcwd()
```
is _current_dir ever used?

Reply:
Yeah, it's used on line 11:

```
    alembic_cfg.set_main_option('script_location', _current_dir)
```

We actually don't want os.getcwd(). That gives the current dir of the user's shell. We need the location of the handler.py file.